src/libs/Navigation/AppNavigator/MainDrawerNavigator.js

```diff
@@ -13,11 +13,12 @@ import ReportScreen from '../../../pages/home/ReportScreen';
 import SidebarScreen from '../../../pages/home/sidebar/SidebarScreen';
 import BaseDrawerNavigator from './BaseDrawerNavigator';
 import * as ReportUtils from '../../ReportUtils';
-import reportPropTypes from '../../../pages/reportPropTypes';
 
 const propTypes = {
     /** Available reports that would be displayed in this navigator */
-    reports: PropTypes.objectOf(reportPropTypes),
+    reports: PropTypes.objectOf(PropTypes.shape({
+        reportID: PropTypes.number,
```

Comment:
Just wondering but why is `reportPropTypes` not used - it should be OK to use those?

Reply:
What i wanted to do here is to document what actually this component needs out of all reportPropTypes. When this is explicit it is much easier to read code and optimize it.

Open for different view @marcaaron